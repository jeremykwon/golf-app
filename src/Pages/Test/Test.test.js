import { render, screen } from '@testing-library/react';
import Test from './Test';

// test('matches snapshot', () => {
//     const utils = render(<Test />);
//     // expect(utils.container).toBeInTheDocument();
//     expect(utils.container).toMatchSnapshot();  // 컴포넌트 수정시 원하는 방식으로 렌더 되는지 비교하기 위한 __snapshots__ 생성
// });

test('shows the props correctly', () => {
    const utils = render(<Test test='test1' />);
    utils.getByText('test1');
});
