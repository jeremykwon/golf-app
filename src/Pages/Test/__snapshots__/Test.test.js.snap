// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <button>
    Test Sound
  </button>
  <p />
</div>
`;
